packages/core/test/financial-templates/common/financial-product-libraries/contract-for-difference-libraries/LinearContractForDiffrenceFinancialProductLibrary.js

```diff
@@ -0,0 +1,99 @@
+const { didContractThrow, ZERO_ADDRESS } = require("@uma/common");
```

Comment:
nit: This file name is mispelled, should be "Difference" not "DIffrence"

Reply:
thank you spell check nick 🤝